ts/packages/ts/src/utils.ts

```diff
@@ -30,3 +31,46 @@ export const TMPL = {
   },
   render: (template: string, context: any) => Mustache.render(template, context, {}, { escape: (value) => value }),
 };
+
+const cfguTypeSet = new Set<CfguType>([
+  'ARN',
+  'AWSRegion',
+  'AZRegion',
+  'AlibabaRegion',
+  'Base64',
+  'Boolean',
+  'Color',
+  'ConnectionString',
+  'Country',
+  'Currency',
+  'DateTime',
+  'DockerImage',
+  'Domain',
+  'Email',
+  'GCPRegion',
+  'Hex',
+  'IBMRegion',
+  'IPv4',
+  'IPv6',
+  'JSONSchema',
+  'Language',
+  'LatLong',
+  'Locale',
+  'MACAddress',
+  'MD5',
+  'MIMEType',
+  'MobilePhone',
+  'MongoId',
+  'Number',
+  'OracleRegion',
+  'RegEx',
+  'SHA',
+  'SemVer',
+  'String',
+  'URL',
+  'UUID',
+]);
+
+export const isStringInCfguType = (input: string): boolean => {
+  return cfguTypeSet.has(input as CfguType);
+};
```

Comment:
Rather than creating a new set, it's better to refer to the `VAL_TYPE` map [here](https://github.com/configu/configu/blob/main/ts/packages/ts/src/ConfigSchema.ts#L21). That way there's a single source of truth which `CfguType`s are supported, as well as the standard being "we support a CfguType if and only if we have a validator hooked up for it".